routing/endpointregistry.go

```diff
@@ -227,6 +230,10 @@ func (r *EndpointRegistry) GetMetrics(hostPort string) Metrics {
 	return e.(*entry)
 }
 
+func (r *EndpointRegistry) MaxUnhealthyEndpointsRatio() float64 {
```

Comment:
Why do we need this to be a part of Endpoint Registry?
I think proxy healthchecker should have access to this value from PassiveHealthCheck config.